.github/workflows/build.yml

```diff
@@ -147,3 +147,6 @@ jobs:
       RUST_BACKTRACE: 1
       RUSTFLAGS: "-D warnings"
       RUSTDOCFLAGS: "-D warnings"
+      # FIXME this is a temporary hack for testing 3.13 prereleases, probably we should have a flag
+      # PYO3_ALLOW_3_13_PRERELEASES or similar so that users don't need to run this flag in their CI
+      UNSAFE_PYO3_SKIP_VERSION_CHECK: "1"
```

Comment:
For now I'd like to merge this PR with this flag set, but I think for users to do real testing with 3.13 we shouldn't ask them to set this.

Reply:
Is the idea that we'd merge this as is, get the tests passing, and then declare support and remove this?

Reply:
Yes, it looks like there are some test cases to fix beyond the FFI changes which I tested locally.

I'm not totally sure what "support" looks like. We don't expect ABI changes any more, but it's possible, and unlike programs using the headers which will compile fine if there's a slight adjustment, we'd have an ABI incompatibility.

Maybe that's fine and we just declare 3.13 support anyway as good enough, or maybe we need an opt-in to 3.13 prereleases and save declaring full 3.13 support until after the release (or the RC).

Reply:
Last beta is ABI stable.

Reply:
I had a quick chat with @hugovk today at PyCon. What I took from that conversation is that, as you say, there should be no ABI breaks, and it's desirable for packages to start uploading PyPI so that downstream code can get testing.

So all in all, the feeling was that we should get on with declaring 3.13 support and the CPython release team are ok with the unlikely possibility that there may be changes which cause breakages of early wheels.

I'll take a look at the test break and see if I can get this PR totally green.